app/models/manageiq/providers/atomic/container_manager.rb

```diff
@@ -0,0 +1,22 @@
+class ManageIQ::Providers::Atomic::ContainerManager < ManageIQ::Providers::ContainerManager
+  include ManageIQ::Providers::Openshift::ContainerManagerMixin
```

Comment:
The other shared mixins live in Kubernetes namespace...why is this one in Openshift?